api/external/cfgmgmt/cfgmgmt.proto

```diff
@@ -316,4 +316,22 @@ service ConfigMgmt {
   rpc NodeExport(cfgmgmt.request.NodeExport) returns (stream chef.automate.api.common.ExportData) {};
 
   rpc ReportExport(cfgmgmt.request.ReportExport) returns (stream chef.automate.api.common.ExportData) {};
+
+  /*
+  GetNodesFieldValueCounts
+
+  Returns a list fields with value counts
+
+  Authorization Action:
+  ```
+  infra:nodes:list
+  ```
+  */
+  rpc GetNodesFieldValueCounts(cfgmgmt.request.NodesFieldValueCounts) returns (cfgmgmt.response.NodesFieldValueCounts) {
+    option (google.api.http).get = "/cfgmgmt/nodes_field_value_counts";
```

Comment:
@chef/ux-team do you have any strong opinions about this api? 

i can't think of anything clearer than `nodes_field_value_counts` but i also kinda stared at it sideways for a min after seeing it

Reply:
i havent been following very closely along with this, but what does field_value mean? is it different than node_counts? or nodes_field_counts?

Reply:
oh i get it now, hmm ill think on this tonight.

Reply:
these are some random thoughts i had about this...
(but i dont have strong opinions)
- i could not look at the endpoint name and understand what it was for
- i think `term` might better work as `value`
- and `field` as `type`
- then maybe `node_metadata_counts` as the name

Reply:
ooooh, i like `node_metadata_counts` !

Reply:
Yeah, I have had a lot of problems naming this function. Also even talking about what it does. I will update it to the above suggestions.